packages/edit-site/src/store/actions.js

```diff
@@ -466,17 +468,14 @@ export const revertTemplate =
 					.dispatch( noticesStore )
 					.createSuccessNotice( __( 'Template reverted.' ), {
 						type: 'snackbar',
+						id: noticeId,
 						actions: [
 							{
 								label: __( 'Undo' ),
 								onClick: undoRevert,
 							},
 						],
 					} );
-			} else {
```

Comment:
this may be needed for some other usage of revert template, but I couldn't see where.